x/application/client/cli/tx_delegate_to_gateway_test.go

```diff
@@ -48,39 +51,35 @@ func TestCLI_DelegateToGateway(t *testing.T) {
 	}{
 		{
 			desc:           "delegate to gateway: valid",
-			appAddress:     appAccount.Address.String(),
-			gatewayAddress: gatewayAccount.Address.String(),
+			appAddress:     appBech32,
```

Comment:
I think we should one of `appAddr` or `appAddrBech32` to make the variable name more semantically clear.